components/ai_chat/core/common/features.cc

```diff
@@ -52,4 +52,12 @@ bool IsContextMenuRewriteInPlaceEnabled() {
   return base::FeatureList::IsEnabled(features::kContextMenuRewriteInPlace);
 }
 
+BASE_FEATURE(kPageContentRefine,
```

Comment:
Why do we use a base feature here and not a feature flag, since this requires the base AIChat feature to be enabled?

Reply:
Good point, I will change that in a follow up PR.

Reply:
hmm actually in this case we still have to use feature flag because we want it to be able to toggle on manually by users instead of random trial to enable it.